ops/ansible/playbooks-ccs/files/client_data_server_ab2d_dev_certificate.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
+MIIEsDCCApgCCQDnd04aeNwuMDANBgkqhkiG9w0BAQsFADAaMRgwFgYDVQQDDA9h
+YjJkLWRldi1jbGllbnQwHhcNMjAwMTIzMTQzOTA0WhcNMjUwMTIxMTQzOTA0WjAa
+MRgwFgYDVQQDDA9hYjJkLWRldi1jbGllbnQwggIiMA0GCSqGSIb3DQEBAQUAA4IC
+DwAwggIKAoICAQCqUNyoKLFJaZhCvmF5WSbV0vKw20+N/OQPVr0MPQ/VusT9nmpJ
+INbzN/LIgCXS1cBp6eUX9rExGiMGFyydc8U32GWqgBU4Fe3FL9GLsa4ouublNcRJ
+U+Q2xn51OOgWX96Zn4dQMe6aLo3pKx79szk6LijK2UkEfP554gQyRAx2/XUK+xoj
+OGGNlMy/1W0SitAtZOPV9UcQf85MT1of1tXIH3hNQyRM0benP3UYJqc/JUwd1owi
+BmdUL3XBBzBCmQeOqt55X1fa6O1E8ixuJWohmA5xx0Lv1AKuwpHEv/gd8Uw60FxJ
+OXM9cPMIHknmI0ZTIduzZwSYe0I2jX9w+f9gicHxAE1P9LLlFhfVTb9gINywmB2B
+1mQ7N5NPI7El01XwK2uJ+2BcVKBqnD6f9ALyBo2xthP+JNbMw8Y/1g0V+UFnAtu9
+eS918qsYdJRBSzBf8P/NEZnsgqaRR3zvHesKJEN5HDtQbvLAe9nCnGOxNKKlkBmc
+nJuoeJxH6VecJy0sgTGIwsCz9RYjIrV2ijRI3SSpgPTMpf5Lg1isUKZAxPceRRnu
+TTkxmmR/7QKr6tZUnqheCm/2/t2NnnC5Osn6Ty4kCxqfYUDZdRsKiEE8OiDBJHJg
+c8Sjx8Q8w4KZst05MUZRzXgzLd+orzn1N8Ml+k2xtNBqhyjMFACNaJLGJwIDAQAB
+MA0GCSqGSIb3DQEBCwUAA4ICAQAhtXAzEbgvGgON9MjzbjUhfOy3R9MXrMBi3L4Y
+/URmkXpcOX1OvJLGWZfz6Wo77w3Sov4YwJUvfjXiPwzGAZuuzjW7gcIQvNKrI2Jp
+bcaTXHtS2WsuQ98guVUJ1jUhl0y3+4e0wZL3PYONHgsmwvKjQB2sQKM7ODnkCBay
+WtFqZGeNTrUxEFXcUEtV9AWeZnU512Gmw9cAmEfQ8LDxc1bwK69Ar4jyagpwfnfs
+uTF/Zr6uGexYZ0IbO+dFo/4CNO6krPA4vTcmX9BP+mIHgxEL0py4tAbf3ZDOunVM
+/RPl01s4ezes/5yomXHrI1LegrIHUKE5JJRh1KRTo/vPQF2Jy50856Qwa2gZh5Pn
+6fAbfJh6tKtuKe7pEuS3Cl6Uhhe4pc7rXnvjia3n8AOb4Z39d9Vss4vDKH2zI0uY
+mfSI4M3pbEJqMisLaHWWUJYTtXTV94qQzamNjSXj3AM2PoDuTgSXDkP8oirvJcUb
+VQAiks62zRB1HW0e+X+w/IOyW9CLSuli/faQPIar8CC5ANspfxYuWaaXwHhm2D4X
+5iDRX2QxPkvY1T/bJh3JLufMYA+QPsXZypdW8x1LBayz5cBHdv8OwDjhhZtw+3cd
+loenVP0hmbq9x/RUwNv1mxPwen0WM0zPrcEl6Rg7rrkUqxHEHoUun0YwUBSAZlO8
+EB/xJA==
+-----END CERTIFICATE-----
```

Comment:
```
openssl x509 -text -in ops/ansible/playbooks-ccs/files/client_data_server_ab2d_dev_certificate.pem
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 16678885620593733168 (0xe7774e1a78dc2e30)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=ab2d-dev-client
        Validity
            Not Before: Jan 23 14:39:04 2020 GMT
            Not After : Jan 21 14:39:04 2025 GMT
        Subject: CN=ab2d-dev-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:aa:50:dc:a8:28:b1:49:69:98:42:be:61:79:59:
                    26:d5:d2:f2:b0:db:4f:8d:fc:e4:0f:56:bd:0c:3d:
                    0f:d5:ba:c4:fd:9e:6a:49:20:d6:f3:37:f2:c8:80:
                    25:d2:d5:c0:69:e9:e5:17:f6:b1:31:1a:23:06:17:
                    2c:9d:73:c5:37:d8:65:aa:80:15:38:15:ed:c5:2f:
                    d1:8b:b1:ae:28:ba:e6:e5:35:c4:49:53:e4:36:c6:
                    7e:75:38:e8:16:5f:de:99:9f:87:50:31:ee:9a:2e:
                    8d:e9:2b:1e:fd:b3:39:3a:2e:28:ca:d9:49:04:7c:
                    fe:79:e2:04:32:44:0c:76:fd:75:0a:fb:1a:23:38:
                    61:8d:94:cc:bf:d5:6d:12:8a:d0:2d:64:e3:d5:f5:
                    47:10:7f:ce:4c:4f:5a:1f:d6:d5:c8:1f:78:4d:43:
                    24:4c:d1:b7:a7:3f:75:18:26:a7:3f:25:4c:1d:d6:
                    8c:22:06:67:54:2f:75:c1:07:30:42:99:07:8e:aa:
                    de:79:5f:57:da:e8:ed:44:f2:2c:6e:25:6a:21:98:
                    0e:71:c7:42:ef:d4:02:ae:c2:91:c4:bf:f8:1d:f1:
                    4c:3a:d0:5c:49:39:73:3d:70:f3:08:1e:49:e6:23:
                    46:53:21:db:b3:67:04:98:7b:42:36:8d:7f:70:f9:
                    ff:60:89:c1:f1:00:4d:4f:f4:b2:e5:16:17:d5:4d:
                    bf:60:20:dc:b0:98:1d:81:d6:64:3b:37:93:4f:23:
                    b1:25:d3:55:f0:2b:6b:89:fb:60:5c:54:a0:6a:9c:
                    3e:9f:f4:02:f2:06:8d:b1:b6:13:fe:24:d6:cc:c3:
                    c6:3f:d6:0d:15:f9:41:67:02:db:bd:79:2f:75:f2:
                    ab:18:74:94:41:4b:30:5f:f0:ff:cd:11:99:ec:82:
                    a6:91:47:7c:ef:1d:eb:0a:24:43:79:1c:3b:50:6e:
                    f2:c0:7b:d9:c2:9c:63:b1:34:a2:a5:90:19:9c:9c:
                    9b:a8:78:9c:47:e9:57:9c:27:2d:2c:81:31:88:c2:
                    c0:b3:f5:16:23:22:b5:76:8a:34:48:dd:24:a9:80:
                    f4:cc:a5:fe:4b:83:58:ac:50:a6:40:c4:f7:1e:45:
                    19:ee:4d:39:31:9a:64:7f:ed:02:ab:ea:d6:54:9e:
                    a8:5e:0a:6f:f6:fe:dd:8d:9e:70:b9:3a:c9:fa:4f:
                    2e:24:0b:1a:9f:61:40:d9:75:1b:0a:88:41:3c:3a:
                    20:c1:24:72:60:73:c4:a3:c7:c4:3c:c3:82:99:b2:
                    dd:39:31:46:51:cd:78:33:2d:df:a8:af:39:f5:37:
                    c3:25:fa:4d:b1:b4:d0:6a:87:28:cc:14:00:8d:68:
                    92:c6:27
                Exponent: 65537 (0x10001)
    Signature Algorithm: sha256WithRSAEncryption
```
There's no private key material here, right? ✅ 
Is the cert different from any other certs from the same project (just watching out for oopses)? ✅ 
Is its Subject unique? ✅ 
Does it have a reasonably useful/informative Subject? ✅ 
Does it have a reasonable key size (>= 2048)? ✅